<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1062</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionConnect_to_MongoDB"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_to_full_extend"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionRefresh_layer"/>
    <addaction name="actionMove"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>Query</string>
    </property>
    <addaction name="action_geoWithin"/>
    <addaction name="action_geoNear"/>
    <addaction name="action_geoIntersects"/>
   </widget>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>Import</string>
    </property>
    <addaction name="actionImport_shapefile_shp"/>
   </widget>
   <addaction name="menuConnect"/>
   <addaction name="menuView"/>
   <addaction name="menuQuery"/>
   <addaction name="menuImport"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionConnect_to_MongoDB"/>
   <addaction name="separator"/>
   <addaction name="actionMove"/>
   <addaction name="actionZoom_to_full_extend"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="actionRefresh_layer"/>
   <addaction name="separator"/>
   <addaction name="action_geoWithin"/>
   <addaction name="action_geoNear"/>
   <addaction name="action_geoIntersects"/>
   <addaction name="separator"/>
   <addaction name="actionImport_shapefile_shp"/>
  </widget>
  <action name="actionConnect_to_MongoDB">
   <property name="icon">
    <iconset>
     <normaloff>icons/dbConnect.png</normaloff>icons/dbConnect.png</iconset>
   </property>
   <property name="text">
    <string>Connect to MongoDB</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>icons/closeApp.png</normaloff>icons/closeApp.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionZoom_to_full_extend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/zoom-extent.png</normaloff>icons/zoom-extent.png</iconset>
   </property>
   <property name="text">
    <string>Zoom to full extend</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/zoom_in.png</normaloff>icons/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/zoom_out.png</normaloff>icons/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionRefresh_layer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/action_refresh.png</normaloff>icons/action_refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh layer</string>
   </property>
  </action>
  <action name="action_geoWithin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/within.png</normaloff>icons/within.png</iconset>
   </property>
   <property name="text">
    <string>$geoWithin</string>
   </property>
  </action>
  <action name="action_geoNear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/circle.png</normaloff>icons/circle.png</iconset>
   </property>
   <property name="text">
    <string>$geoNear</string>
   </property>
  </action>
  <action name="action_geoIntersects">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/intersect.png</normaloff>icons/intersect.png</iconset>
   </property>
   <property name="text">
    <string>$geoIntersects</string>
   </property>
  </action>
  <action name="actionImport_shapefile_shp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/importData.png</normaloff>icons/importData.png</iconset>
   </property>
   <property name="text">
    <string>Import shapefile(*.shp)</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/move.png</normaloff>icons/move.png</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
